PennMobile/Dining/Controllers/DiningViewController.swift

```diff
@@ -35,13 +35,17 @@ class DiningViewController: GenericTableViewController {
         super.viewWillAppear(animated)
         fetchDiningHours()
         
-        if viewModel.shouldShowDiningBalances {
+        UserDefaults.standard.set(hasDiningPlan: false)
```

Comment:
are we explicitly setting this to false and then immediately doing an if check?

Reply:
Oops meant to remove that.